src/test/java/com/snowflake/kafka/connector/ConnectorConfigValidatorTest.java

```diff
@@ -838,6 +838,53 @@ public void testEnableStreamingChannelMigrationConfig_invalidWithSnowpipe() {
             SnowflakeSinkConnectorConfig.ENABLE_CHANNEL_OFFSET_TOKEN_MIGRATION_CONFIG);
   }
 
+  @Test
+  public void testEnableStreamingChannelOffsetVerificationConfig() {
+    Map<String, String> config = getConfig();
```

Comment:
I have recently added a proper builder for configs. Please use `SnowflakeSinkConnectorConfigBuilder.snowpipeConfig().build()` instead.

Reply:
Fixed with Artur's approval.